tests/integration/annotation_import/test_send_to_annotate_mea.py

```diff
@@ -32,3 +56,7 @@ def test_send_to_annotate_from_model(client, configured_project,
     destination_data_rows = list(destination_batches[0].export_data_rows())
     assert len(destination_data_rows) == len(data_row_ids)
     assert all([dr.uid in data_row_ids for dr in destination_data_rows])
+
+    # Since data rows were added to a review queue, predictions should be imported into the project as labels
+    destination_project_labels = (list(destination_project.labels()))
+    assert len(destination_project_labels) == len(data_row_ids)
```

Comment:
Possible also to compare some label attributes? Also to make sure len(...) is not 0?